rules/php53/src/Rector/Assign/ClearReturnNewByReferenceRector.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php53\Rector\Assign;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Assign;
+use PhpParser\Node\Expr\AssignRef;
+use PhpParser\Node\Expr\New_;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * @sponsor Thanks https://twitter.com/afilina for sponsoring this rule
+
+ * @see https://3v4l.org/UJN6H
+ * @see \Rector\Php53\Rector\Assign\ClearReturnNewByReferenceRector
+ */
+final class ClearReturnNewByReferenceRector extends AbstractRector
+{
```

Comment:
Hi @afilina,

we have this habit in Rector, that any person/company that sponsors more rules that are useful and shared with community,
will get this "sponsor badge" above the rule as a small "thank you" for doing so.

Is this line ok? We can link also website. e.g.

```php
* @sponsor Thanks https://twitter.com/afilina for sponsoring this rule - visit her on https://www.youtube.com/c/AnnaFilina
```